portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -118,9 +119,13 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
         // Get the input from the form and log time of comment
         String commentText = request.getParameter(COMMENT_FORM_PARAMETER);
         long timestamp = System.currentTimeMillis();
-        String imageURL = getUploadedFileUrl(request, "image");
-        System.out.println(imageURL);
-
+        try {
+            String imageURL = getUploadedFileUrl(request);
+        } catch(IOException e) {
+            response.sendError(HttpServletResponse.SC_BAD_REQUEST, 
+              e.toString());
```

Comment:
When this happens, will it continue to execute the rest of the function? Should you `return;` after this? Same with the other places.

Reply:
Yes I just added the return statements